src/components/expression/expression-renderer.js

```diff
@@ -310,15 +373,33 @@ genotet.ExpressionRenderer.prototype.drawMatrixCells_ = function() {
       .range(genotet.data.redYellowScale);
   }
   var transformLeft = 0;
-  var transformTop = this.conditionLabelHeight_ + this.profileHeight_;
+  var transformTop = this.profileHeight_;
   if (this.data.options.showGeneLabels) {
-    transformLeft += this.LABEL_MARGIN_;
-  }
-  else if (this.data.options.showProfiles) {
+    transformLeft += this.LABEL_MARGIN;
+  } else if (this.data.options.showProfiles) {
     transformLeft += this.DEFAULT_PROFILE_MARGIN;
   }
   if (this.data.options.showConditionLabels) {
-    transformTop += this.LABEL_MARGIN_;
+    transformTop += this.conditionLabelHeight_ +
+      this.LABEL_MARGIN + this.LABEL_DIFFERENCE;
+  }
+
```

Comment:
transformLeft and transformTop are applied to every heatmapRow. Why not applying them to this.svgHeatmapContent_ instead? Also this could be written in layout().